flowspec.runtime/src/main/java/mb/flowspec/terms/IStrategoAppl2.java

```diff
@@ -7,10 +7,7 @@
 
 import javax.annotation.Nullable;
 
-import org.spoofax.interpreter.terms.IStrategoAppl;
-import org.spoofax.interpreter.terms.IStrategoList;
-import org.spoofax.interpreter.terms.IStrategoTerm;
-import org.spoofax.interpreter.terms.ITermPrinter;
+import org.spoofax.interpreter.terms.*;
```

Comment:
I thought we didn't do star imports in Spoofax code. Pretty sure that's in the style files... You may need to configure that in IntelliJ separately though

Reply:
Yup, I seem to remember something similar. Apparently the only way to configure this in IntelliJ is to put a high number for the number of imports to collapse in a star import, say, 999. :shrug: Fixed.